### PR TITLE
fix(console): only show enabled connectors in table

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Only show enabled connectors (platforms) in table.

## Linear Issue Reference

LOG-3103
LOG-3078
LOG-3090

## Testing

- [x] Alipay only enabled web.
- [x] GitHub is disabled (removed).

截屏2022-06-20 下午1 32 48

### CODE REVIEW

File: packages/console/src/pages/Connectors/components/ConnectorName/index.tsx [hunk 1]
```diff
@@ -23,7 +23,8 @@ type Props = {
 
 const ConnectorName = ({ type, connectors, onClickSetup }: Props) => {
   const { t } = useTranslation(undefined);
-  const connector = connectors[0];
+  const enableConnectors = connectors.filter(({ enabled }) => enabled);
```
Comment (gao-sun) [inline packages/console/src/pages/Connectors/components/ConnectorName/index.tsx:26]: `enabledConnectors`

**Review by gao-sun — APPROVED** [review]

File: packages/console/src/pages/Connectors/index.tsx [hunk 2]
```diff
@@ -48,7 +48,9 @@ const Connectors = () => {
       return;
     }
 
-    return data?.filter((connector) => connector.type === ConnectorType.Social);
+    return data?.filter(
+      (connector) => connector.enabled && connector.type === ConnectorType.Social
```
Comment (demonzoo) [inline packages/console/src/pages/Connectors/index.tsx:52]: nit: deconstruct first

**Review by demonzoo — APPROVED** [review]

File: packages/console/src/pages/Connectors/components/ConnectorName/index.tsx (hunk 1 above)
Comment (wangsijie) [reply packages/console/src/pages/Connectors/components/ConnectorName/index.tsx]: updated

File: packages/console/src/pages/Connectors/index.tsx (hunk 2 above)
Comment (wangsijie) [reply packages/console/src/pages/Connectors/index.tsx]: updated

**Pull request merged**